File: 9041/lab07/count_word.pl
#!/usr/bin/perl -w


$word = $ARGV[0];

while ($line = <STDIN>) {
	chomp $line;
 	@content = split /[^a-zA-Z]/,$line;
	foreach $str (@content){
		if (lc $str eq lc $word) {
		   
		   push @result, $str;		
		}
	}
	
}
$out = $#result + 1; 
print "$word occurred $out times\n"; 

File: 9041/lab07/count_word.py
#!/usr/bin/python3

import sys
import re
total = 0
target = sys.argv[1]
for line in sys.stdin:
    for char in re.split('[^a-zA-Z]', line):
        if (char.lower() == target.lower()):
            total += 1
            
print(target, "occurred", total, "times")
